client/src/main/java/org/apache/falcon/client/FalconClient.java

```diff
@@ -84,57 +84,55 @@
 
     public static final String WS_HEADER_PREFIX = "header:";
     public static final String USER = System.getProperty("user.name");
-    public static final String AUTH_URL = "api/options?" + PseudoAuthenticator.USER_NAME + "=" + USER;
+    private static final String AUTH_URL = "api/options?" + PseudoAuthenticator.USER_NAME + "=" + USER;
 
-
-
-    public static final String PATH = "path";
-    public static final String COLO = "colo";
+    private static final String PATH = "path";
+    private static final String COLO = "colo";
     private static final String KEY = "key";
     private static final String VALUE = "value";
     public static final String CLUSTER = "cluster";
-    public static final String RUN_ID = "runid";
-    public static final String FORCE = "force";
-    public static final String SHOW_SCHEDULER = "showScheduler";
-    public static final String ENTITY_NAME = "name";
-    public static final String ENTITY_TYPE = "type";
-    public static final String SKIP_DRYRUN = "skipDryRun";
-    public static final String FILTER_BY = "filterBy";
-    public static final String ORDER_BY = "orderBy";
-    public static final String SORT_ORDER = "sortOrder";
-    public static final String OFFSET = "offset";
-    public static final String NUM_RESULTS = "numResults";
-    public static final String START = "start";
-    public static final String END = "end";
-    public static final String INSTANCE_TIME = "instanceTime";
-    public static final String INSTANCE_STATUS = "instanceStatus";
-    public static final String PROPERTIES = "properties";
+    private static final String RUN_ID = "runid";
```

Comment:
Nit: Why has the access been changed? Ideally, these should be the ones used by FalconCLI too, rather than having these constants repeated FalconCLIConstants.